#25: Made ParametersMapper public (#26)
exasol/error-reporting-java: doc/changes/changes_0.4.0.md

# error-reporting-java 0.4.0, released 2021-03-10

> Code Name: Added PlaceholderMatcher to public API

Reviewer: ```suggestion
Code Name: Added `PlaceholderMatcher` to public API
```

## Summary

> In this release we made PlaceholderMatcher public so that it can be used in the error-code-crawler.

Reviewer: ```suggestion
In this release we made the `PlaceholderMatcher` public so that it can be used in other projects.
```

Author: ```suggestion
In this release we made the `PlaceholderMatcher` public so that it can be used in the error-code-crawler.
```

Author: I think it's good to give some reasons why we did a change. Cyclic dependencies in free text should not be an issue ;-)
